com.unity.uiwidgets/Runtime/widgets/text.cs

```diff
@@ -91,17 +89,15 @@ public class DefaultTextStyle : InheritedTheme {
             return context.dependOnInheritedWidgetOfExactType<DefaultTextStyle>() ?? new DefaultTextStyle();
         }
         public override bool updateShouldNotify(InheritedWidget oldWidget) {
-            //InheritedWidget
-            //DefaultTextStyle
+            
             oldWidget = (DefaultTextStyle) oldWidget;
-           
             return style !=  ((DefaultTextStyle)oldWidget).style ||
-                textAlign !=  ((DefaultTextStyle)oldWidget).textAlign ||
-                softWrap !=  ((DefaultTextStyle)oldWidget).softWrap ||
-                overflow !=  ((DefaultTextStyle)oldWidget).overflow ||
-                maxLines !=  ((DefaultTextStyle)oldWidget).maxLines ||
-                textWidthBasis !=  ((DefaultTextStyle)oldWidget).textWidthBasis ||
-                textHeightBehavior !=  ((DefaultTextStyle)oldWidget).textHeightBehavior;
+                   textAlign !=  ((DefaultTextStyle)oldWidget).textAlign ||
```

Comment:
we can cast oldWidget to a DefaultTextStyle widget (calling it _oldWidget) and use the latter here. e.g., textAlign != _oldWidget.textAlign.